bullet_train-themes-light/lib/tasks/application.rb

```diff
@@ -19,14 +27,17 @@ def self.eject_theme(theme_name, ejected_theme_name)
         %x(sed -i #{'""' if `echo $OSTYPE`.include?("darwin")} "s/#{theme_name}/#{ejected_theme_name}/g" #{Rails.root}/tailwind.mailer.#{ejected_theme_name}.config.js)
 
         puts "Ejecting stylesheets into `./app/assets/stylesheets/#{ejected_theme_name}`."
-        `mkdir #{Rails.root}/app/assets/stylesheets`
+        `mkdir -p #{Rails.root}/app/assets/stylesheets`
```

Comment:
We can do the `mkpath` trick here too:

```suggestion
        Rails.root.join("app/assets/stylesheets").mkpath
```